docs/release-notes/v11.3.0.md

```diff
@@ -0,0 +1,179 @@
+---
+id: v11.3.0
+title: Version v11.3.0 Release Notes
+sidebar_label: v11.3.0
+image: "img/release-note-link-preview.png"
+---
+
+_June 22th, 2023_
+
+## Console
+
+### New Features
+
+#### Manage and configure Webhooks directly via Console UI
+
+Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
+Whereas before today this was only possible through Console CMS, now the user can enter the "Company Overview" area and under the "Webhooks" section can easily and intuitively manage all the respective hooks.
```

Comment:
```suggestion
#### Manage and configure Webhooks directly via Console UI

Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
Whereas before today this was only possible through Console CMS, now the user can enter the Company Overview area and easily manage all the respective hooks under the new "Webhooks" section.
```